test/e2e/rules_api_test.go

```diff
@@ -87,7 +104,7 @@ func TestRulesAPI_Fanout(t *testing.T) {
 	ruleAndAssert(t, ctx, q.HTTPEndpoint(), "", []*rulespb.RuleGroup{
 		{
 			Name: "example_abort",
-			File: "/shared/rules/rules-0.yaml",
+			File: "/shared/rules/rules.yaml",
```

Comment:
weird...

Reply:
I separate the rule files for Prometheus and Thanos ruler. https://github.com/thanos-io/thanos/pull/4135/files#diff-043bcd9734d5ba38687c9b0199a81e047ff6f744e1b5351e6a3c9c8e96f446a9R42

Reply:
Ah, makes sense, Thanks